api/src/libs/request.ts

```diff
@@ -125,7 +146,14 @@ export const processProfileContactsEdit = async (request: Request): Promise<void
             }
         }
 
-        // TODO(sb): implement functionality to delete a contact if a project goes from 2 TL's -> 1 TL.
+        // functionality to delete a contact if a project goes from 2 TL's -> 1 TL.
+        const removeExistingContact = currentContacts.filter(comparerContact(contacts));
+        //remove contact
+      if(removeExistingContact[0]) {
```

Comment:
Magic number. I believe it is better practice to use something like `.pop()` to extract the first value.

Reply:
updated to use map loop instead of if loop